packages/rocketchat-ui-message/message/renderMessageBody.js

```diff
@@ -21,6 +21,36 @@ renderMessageBody = function(msg) {
 	}
 
 	msg.html = message.html.replace(/\n/gm, '<br/>');
+	
+	if(singleEmojiCheck(msg))
+		msg.html = (msg.html).replace("emojione ", "big_emojione ");
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Mixed double and single quotes.](https://www.codacy.com/app/RocketChat/Rocket-Chat/file/2486979418/issues/source?bid=3199634&fileBranchId=3266617#l26)